### PR TITLE
chore: rename makefile release-src to release

### PR DESCRIPTION
**Pull request by gxthrj**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance

- Related issues

___

### CODE REVIEW

**Review by membphis — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -89,7 +89,7 @@ endif
 	.actions/openwhisk-utilities/scancode/scanCode.py --config .actions/ASF-Release.cfg ./
 
 
-.PHONY: release-src
-release-src:
+.PHONY: release
```
Comment (juzhiyuan) [inline Makefile:92]: Does release only do a package work? 

cc @moonming @ShiningRush

Comment (gxthrj) [reply Makefile:92]: Do you want to include the build process?

**Review by moonming — CHANGES_REQUESTED** [review]
please refer to apisix's make release

**Comment by gxthrj** [comment]
Add sha512 and mv the pkgs to release directory .
Refer to apisix's make release.

File: Makefile [hunk 2]
```diff
@@ -91,5 +93,10 @@ endif
 
 .PHONY: release
 release:
-	tar –cf dashboard.tar ./output/*
+	tar -zcvf $(RELEASE).tgz ./output/*
```
Comment (moonming) [inline Makefile:96]: NO.
`release` is ONLY for release manager, who ONLY release source code packge.

**Review by moonming — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (gxthrj) [reply Makefile:96]: Ok, It seems that `make release` should not be used here.
The `output` directory is not source code. They are the result of compilation, including manager and web.
Maybe `make publish` is more suitable. Ref to some web projects , Such as elementUI.

File: .gitignore [hunk 3]
```diff
@@ -42,6 +42,9 @@ build
 manager-api
 
 output
+publish
```
Comment (juzhiyuan) [inline .gitignore:45]: We use `publish` as a folder is not a good idea, can you provide some examples of other popular projects?

The act of ordering this line is to package the product, and we need to think about whether we need to do this here. 

If we need a package as the final product, then just packaged it in the build process.
If we use a package just to let users move it easily, let's call it `make package`, but I don't think it's needed and will take the confusion with `make build`.

cc @moonming @membphis any ideas?

**Review by juzhiyuan — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 3 above)
Comment (moonming) [reply .gitignore:45]: agreed. @gxthrj please refer to other repos first.
To be honest, I don’t know what PR does

Comment (gxthrj) [reply .gitignore:45]: It is for the issue #787 
It seems that I didn't understood the meaning of issue.

**Comment by gxthrj** [comment]
Close the PR,  #816 has been merged. 
It seems no necessary to package the output directory.

**Pull request closed without merging**